rust/apple-client-ffi/build-rust.sh

```diff
@@ -1,117 +0,0 @@
-#!/bin/bash
-
-##################################################
-# We call this from an Xcode run script.
-##################################################
-
-set -euo pipefail
-
-cmd=${1:-""}
-
-# Sanitize the environment to prevent Xcode's shenanigans from leaking
```

Comment:
So these were added as the result of a long debugging session that uncovered some kind of linking issue between iOS libs and rustc (might have been `ring` IIRC?).

Would definitely love to remove these but just adding this note here as a reminder to test both debug/release builds on iOS. I'll make sure to do this as well.